src/libs/actions/App.js

```diff
@@ -1,5 +1,3 @@
-import moment from 'moment-timezone';
-import 'moment/locale/es';
```

Comment:
This should not be removed until we completely deprecate moment library.
reason: https://github.com/Expensify/App/issues/26719#issuecomment-1708873889